ts/types/WebviewMessage.ts

```diff
@@ -22,6 +22,43 @@ export const AlertPayload = t.intersection(
   "AlertPayload"
 );
 
+export const ToastContentR = t.interface({
+  text: t.string
+});
+
+export const ToastContentO = t.partial({
+  type: t.union([
+    t.literal("danger"),
+    t.literal("success"),
+    t.literal("warning")
+  ]),
+  position: t.union([
```

Comment:
When displaying a toast with `center` position, it appears on top, underneath the notch (on the iPhone 14 simulator) while the `top` position places it below the notch, with no margin at all.
Toast's aesthetics are not part of this PR but what do you think about not exposing options that do not render properly? I am suggesting that we remove the `position` parameter and make it appear fixed on bottom